egs/wsj/s5/utils/data/perturb_speed_to_allowed_lengths.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Johns Hopkins University (author: Hossein Hadian)
+# Apache 2.0
+
+
+""" This script perturbs speeds of utterances to force their lengths to some allowed
+    lengths spaced by a factor (like 10%)
+"""
+
+import argparse
+import os
+import sys
+import copy
+import math
+
+parser = argparse.ArgumentParser(description="""This script copies the 'srcdir'
```

Comment:
This script can be modified to follow the normal conventions like in train.py using main() and get_args(). Google standards does not allow global variables in general etc.

Reply:
will fix this (and other scripts).